src/bin/pg_autoctl/monitor.c

```diff
@@ -346,6 +358,266 @@ monitor_node_active(Monitor *monitor,
 }
 
 
+/*
+ * monitor_set_node_candidate_priority updates the monitor on the changes
+ * in the node candidate priority.
+ */
+bool
+monitor_set_node_candidate_priority(Monitor *monitor, int nodeid, char* nodeName, int nodePort, int candidate_priority)
+{
+	PGSQL *pgsql = &monitor->pgsql;
+	const char *sql = "SELECT * FROM pgautofailover.set_node_candidate_priority($1, $2, $3, $4)";
```

Comment:
The server-side function `RETURNS bool`, which is a scalar value, not a composite, not a table. We should directly `SELECT pgautofailover.set_node_candidate_priority($1, $2, $3, $4)` I think.